mysite/__init__.py

```diff
@@ -0,0 +1 @@
+"""Django website skeleton."""
```

Comment:
Well, I think the skeleton is imprecise. While it is one, it's just a temporary state. Here it's just a package. So I'd drop that last word now.